janus_server/src/aggregator.rs

```diff
@@ -95,28 +95,28 @@ pub enum Error {
     #[error("invalid message: {0}")]
     Message(#[from] janus_core::message::Error),
     /// Corresponds to `reportTooLate`, §3.1
-    #[error("task {1:?}: stale report: {0}")]
-    ReportTooLate(ReportMetadata, TaskId),
+    #[error("task {1}: stale report: {0}")]
+    ReportTooLate(Nonce, TaskId),
     /// Corresponds to `reportTooEarly`, §3.1. A report was rejected becuase the timestamp is too far in the future, §4.3.4.
-    #[error("task {1:?}: report from the future: {0}")]
-    ReportTooEarly(ReportMetadata, TaskId),
+    #[error("task {1}: report from the future: {0}")]
+    ReportTooEarly(Nonce, TaskId),
```

Comment:
I think it'd be nice to have the report timestamp in here, which will then show up in logs.

Reply:
👍🏻 While I was touching this code again, I moved the `TaskId` parameter to be first in all cases -- it's always rendered first, and conceptually it's the "largest-scoped" piece of information included in these errors (and I generally like specifying identifiers from largest to smallest scope)